src/AppInstallerCommonCore/Manifest/MsixManifestValidation.cpp

```diff
@@ -36,6 +36,27 @@ namespace AppInstaller::Manifest
         return errors;
     }
 
+    void MsixManifestValidation::Cleanup()
+    {
+        // Clean cache
+        AICLI_LOG(Core, Info, << "Clearing Msix info cache");
+        m_msixInfoCache.clear();
```

Comment:
>m_msixInfoCache.clear(); [](http://example.com/codeflow?start=8&length=24)

This is probably not needed, they are cleared during destruction.

And shall we make this cleanup() method just the destructor, I don't see it being reused. This way caller don't have to remember to call cleanup